_config.yml

```diff
@@ -1,12 +1,14 @@
-plugins:
-  - jekyll-relative-links
-relative_links:
-  enabled: true
-  collections: true
```

Comment:
the `relative-links` plugin (like many others) is enabled by default by github pages, so this is unnecessary